test/blackbox-tests/test-cases/melange/merlin.t

```diff
@@ -37,3 +45,8 @@
   $ dune build @check
   $ dune ocaml-merlin --dump-config="$PWD" | grep -i "$target"
     $TESTCASE_ROOT/_build/default/.output.mobjs/melange)
+
+The melange.emit entry contains a ppx directive
+
+  $ dune ocaml-merlin --dump-config="$(pwd)" | grep -i "ppx"
```

Comment:
I change the command to `dune ocaml merlin dump-config $PWD` in the main branch

Reply:
https://github.com/ocaml/dune/pull/6476/commits/4acd9e1e11639e22b808239e38d1e92108146147